packages/strapi-design-system/src/DatePicker/DatePicker.js

```diff
@@ -64,7 +64,7 @@ export const DatePicker = ({
           <DatePickerButton
             ref={datePickerButtonRef}
             onClick={toggleVisibility}
-            aria-label={selectedDate ? selectedDateLabel(formatDate(selectedDate, locale)) : label}
+            aria-label={selectedDate ? selectedDateLabel(formatDate(selectedDate, locale)) : label || ariaLabel}
```

Comment:
I know it seems strange but I had to add the ariaLabel just for the DateTimePicker case, before we used only the label value

Reply:
I mean doing this, because if you have an aria-label it won't be selected unless there is not label

```suggestion
            aria-label={selectedDate ? selectedDateLabel(formatDate(selectedDate, locale)) : ariaLabel || label}
```